candi.sh

```diff
@@ -349,6 +349,16 @@ package_fetch () {
             cd ${EXTRACTSTO}
             git checkout ${VERSION} --force
             quit_if_fail "candi: git checkout ${VERSION} --force failed"
+            cd ..
+        fi
+        
+        # git cherry-pick on commits given by ${CHERRYPICKCOMMITS}
+        if [ ${STABLE_BUILD} = true ] && [ ! -z "${CHERRYPICKCOMMITS}" ]; then
```

Comment:
Note: I'm not sure if we should place the application of git cherry-pick after "unpacking" in the git trunc.
Another (and may be better opportunity) would be to place it under `apply_(specific)_patches()`, but this function might not be implemented by default.
Any comment on this is helpful for the discussion.